### ISSUE TITLE
Doesn't work on Brave

### ISSUE THREAD

**Issue by rudolphos** [snowie2000/mactype#524]
I have a problem where Brave and Brave Dev is the only chromium browser where mactype doesn't work. It works on Chrome, Vivaldi, Opera, Cent Browser, but not on Brave.

I've tried reinstalling, switching between registry and service modes, nothing has worked.

![image](https://user-images.githubusercontent.com/4366810/55231982-cc3b7c80-522c-11e9-8cd0-08a890d80710.png)

![image](https://user-images.githubusercontent.com/4366810/55231961-bded6080-522c-11e9-906e-21ddb8fe3044.png)

![image](https://user-images.githubusercontent.com/4366810/55239707-ecc10200-523f-11e9-98c2-b92efa428be5.png)

Brave
![image](https://user-images.githubusercontent.com/4366810/56232984-b7fbda00-608a-11e9-8b79-a3efb54fdec7.png)

Chrome
![image](https://user-images.githubusercontent.com/4366810/56232991-bcc08e00-608a-11e9-9154-98a28a1a4506.png)

Brave version: `Version 0.64.47 Chromium: 74.0.3729.61 (Official Build) dev (64-bit)`

**Comment by sammilucia** [comment]
@rudolphos are you using the current Beta5?

**Comment by rudolphos** [comment]
@sammilucia yes. I even reinstalled it yesterday, but still having the problem.

**Comment by sammilucia** [comment]
@rudolphos hmmmm ... thank you. MacType hooks DirectWrite and configures it, I'm wondering if Brave (and Chrome 74?) are just doing their own rendering now. They've been getting closer and closer... @snowie2000 thoughts?

**Comment by Xjs** [comment]
I'm having the same issue, works in Chrome `Version 75.0.3770.90 (Official Build) (64-bit)`, but not in Brave `Version 0.67.83 Chromium: 75.0.3770.87 (Official Build) dev (64-bit)`.

**Comment by JonnyRedHed** [comment]
I'm seeing the same issue with Brave Dev as well.  Win7.

Having to resort to the fudge method of using this extension '_Chrome Font Rendering Enhancer
0.0.5_'.  Which is kinda ok.

**Comment by snowie2000** [comment]
![QQ截图20190619112544](https://user-images.githubusercontent.com/11767189/59734808-70115300-9285-11e9-8ffb-da7c23846913.png)
Brave dev definitely works in Windows7, I haven't tested it in Windows10 though.

**Comment by JonnyRedHed** [comment]
Is there some command line to add to the Brave Dev shortcut, or other setting to make brave dev work on Win 7 x64.  Something I'm missing.

**Comment by rudolphos** [comment]
@snowie2000 now that you said it, I tested it too.

Weird, because mactype works when Brave Dev has been launched with Windows 7 compatibility mode, but doesn't work when launched with Win 8.

![image](https://user-images.githubusercontent.com/4366810/59882712-acdf6600-93bb-11e9-91e9-71559bffe3ef.png)

The only problem is that dark mode won't work, title bar will stay the same color as windows scheme color.

**Comment by Xjs** [comment]
With Version 0.68.83 Chromium: 75.0.3770.100 (Official Build) dev (64-bit), MacType suddenly started working in Brave (Dev).
